### PR TITLE
Update LICENSE for 2201.6.2

### PR DESCRIPTION
**Pull request by tharindu-nw**
## Purpose
$subject

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: ballerina/LICENSE [hunk 1]
```diff
@@ -12,175 +12,175 @@ At the bottom of this file is a table that shows what each license indicated
...
 _________________________________________________________________________________________________________________________________________________
 ballerina-command-1.3.15.jar                                                                        jar            apache2             
 commons-text-1.10.0.jar                                                                             bundle         apache2             
 compiler-0.8.9.jar                                                                                  jar            apache2             
-stream-2201.6.1.jar                                                                                 jar            apache2             
-jballerina.java-2201.6.1.jar                                                                        jar            apache2             
-gson-2.8.9.jar                                                                                      jar            apache2             
+future-2201.6.2-20230907-231100-3134cfa2.jar                                                        jar            apache2
```
Comment (HindujaB) [inline ballerina/LICENSE:20]: Shall we remove the timestamps here and the other places?

Comment (tharindu-nw) [reply ballerina/LICENSE]: Removed in https://github.com/ballerina-platform/ballerina-distribution/pull/4979/commits/8eaa73fd8613d6b0cd81f5883045036a9c4121f4

File: ballerina/LICENSE [hunk 2]
```diff
@@ -12,175 +12,175 @@ At the bottom of this file is a table that shows what each license indicated
...
 _________________________________________________________________________________________________________________________________________________
 ballerina-command-1.3.15.jar                                                                        jar            apache2             
 commons-text-1.10.0.jar                                                                             bundle         apache2             
 compiler-0.8.9.jar                                                                                  jar            apache2             
-stream-2201.6.1.jar                                                                                 jar            apache2             
-jballerina.java-2201.6.1.jar                                                                        jar            apache2             
-gson-2.8.9.jar                                                                                      jar            apache2             
+future-2201.6.2.jar                                                        jar            apache2
```
Comment (HindujaB) [inline ballerina/LICENSE:20]: shall we align the `jar` and `apache2` parts with other lines?

Comment (tharindu-nw) [reply ballerina/LICENSE]: Fixed in https://github.com/ballerina-platform/ballerina-distribution/pull/4979/commits/e7af3b1cb1930e4b240bd1d988e3eebddb7d5aa8

**Review by HindujaB — APPROVED** [review]
LGTM

**Pull request merged**